charts/harvester-vm-import-controller/values.yaml

```diff
@@ -11,8 +11,8 @@ image:
   tag: main-head
 
 imagePullSecrets: []
-nameOverride: ""
-fullnameOverride: ""
+nameOverride: "harvester-vm-import-controller"
```

Comment:
Using fullnameOverride seems to be enough.

Reply:
yep.. fixed it.